main.tf

```diff
@@ -2049,6 +2049,138 @@ module "secrets_store_csi_driver" {
+
+  # IAM role for service account (IRSA)
+  set_irsa_names = [
+    "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn",
+  ]
+  create_role                   = try(var.aws_for_fluentbit.create_role, true)
+  role_name                     = try(var.aws_for_fluentbit.role_name, "aws-for-fluent-bit")
+  role_name_use_prefix          = try(var.aws_for_fluentbit.role_name_use_prefix, true)
+  role_path                     = try(var.aws_for_fluentbit.role_path, "/")
+  role_permissions_boundary_arn = lookup(var.aws_for_fluentbit, "role_permissions_boundary_arn", null)
+  role_description              = try(var.aws_for_fluentbit.role_description, "IRSA for aws-for-fluent-bit")
+  role_policies                 = lookup(var.aws_for_fluentbit, "role_policies", {})
+
+  source_policy_documents = compact(concat(
+    data.aws_iam_policy_document.aws_for_fluentbit[*].json,
+    lookup(var.aws_for_fluentbit, "source_policy_documents", [])
+  ))
+  override_policy_documents = lookup(var.aws_for_fluentbit, "override_policy_documents", [])
+  policy_statements         = lookup(var.aws_for_fluentbit, "policy_statements", [])
+  policy_name               = try(var.aws_for_fluentbit.policy_name, "aws-for-fluent-bit")
+  policy_name_use_prefix    = try(var.aws_for_fluentbit.policy_name_use_prefix, true)
+  policy_path               = try(var.aws_for_fluentbit.policy_path, null)
+  policy_description        = try(var.aws_for_fluentbit.policy_description, "IAM Policy for AWS Fluentbit")
+
+  oidc_providers = {
+    this = {
+      provider_arn = var.oidc_provider_arn
+      # namespace is inherited from chart
+      service_account = local.aws_for_fluentbit_service_account
+    }
+  }
+
+  tags = var.tags
+}
+
+resource "aws_cloudwatch_log_group" "aws_for_fluentbit" {
+  count = try(var.aws_for_fluentbit_cw_log_group.create, true) && var.enable_aws_for_fluentbit ? 1 : 0
+
+  name              = try(var.aws_for_fluentbit_cw_log_group.name, null)
+  name_prefix       = try(var.aws_for_fluentbit_cw_log_group.name_prefix, "/${var.cluster_name}/aws-fluentbit-logs")
+  retention_in_days = try(var.aws_for_fluentbit_cw_log_group.retention, 90)
+  kms_key_id        = try(var.aws_for_fluentbit_cw_log_group.kms_key_arn, null)
+  skip_destroy      = try(var.aws_for_fluentbit_cw_log_group.skip_destroy, false)
+  tags              = merge(var.tags, try(var.aws_for_fluentbit_cw_log_group.tags, {}))
+}
+
+data "aws_iam_policy_document" "aws_for_fluentbit" {
+  count = try(var.aws_for_fluentbit_cw_log_group.create, true) && var.enable_aws_for_fluentbit ? 1 : 0
+  statement {
+    sid       = "PutLogEvents"
+    effect    = "Allow"
+    resources = ["arn:${local.partition}:logs:${local.region}:${local.account_id}:log-group:*:log-stream:*"]
+    actions = [
+      "logs:PutLogEvents"
+    ]
+  }
+
+  statement {
+    sid       = "CreateCWLogs"
+    effect    = "Allow"
+    resources = ["arn:${local.partition}:logs:${local.region}:${local.account_id}:log-group:*"]
```

Comment:
Can we try for `aws_for_fluentbit_cw_log_group.name` and then fall back to `*`?

Reply:
It should be already there 😅.